test/bitbucket.js

```diff
@@ -294,4 +557,19 @@ describe('/POST bitbucket', () => {
                 done();
             });
     });
-});
+
+    it('issue:updated', (done) => {
+        chai.request(server)
+            .post('/api/webhooks/test/test/bitbucket')
+            .set("X-Event-Key", "issue:updated")
+            .set("test", "true")
+            .send(issueUpdatedJson)
+            .end((err, res) => {
+                res.should.have.status(200);
+                console.log(res.body);
+                res.body.should.be.a('object');
+                res.body.should.have.property('embeds');
+                done();
+            });
+    });
+});
```

Comment:
It might be best to remove the additional test added by this PR. So far every provider has only one test, even though it may have many action types defined. Plus, the testing process is just to ensure that the payload is loosely in the correct format, and build purposes. It doesn't actually send any data, but makes sure the provider is functioning for the given payload. One of these such tests should be sufficient.